### DISCUSSION TITLE
How do I toggle my explorer in VSCode?

### DISCUSSION THREAD

**Discussion by iakuraa** [vscode-neovim/vscode-neovim#2073] [Q&A] [answered]
How do I do it in Neovim without any configuration? I know Ctrl + B is the VSCode shortcut, but it does not work in the editor itself. So what is the vanilla binding for Neovim to open the explorer?

**Comment by ollien** [comment, marked as answer]
I do not believe there is anything built-in for this. However, if you'ore willing to do configuration yourself, you have a couple of options

1. Remove "b" from [`ctrlKeysForNormalMode`](https://github.com/vscode-neovim/vscode-neovim?tab=readme-ov-file#disable-keybindings)
2. Add a shortcut in your keybindings.json

```json5
	{
		"command": "workbench.action.toggleSidebarVisibility",
		"key": "ctrl+alt+b", // or whatever key combo you want
		"when": "neovim.init && neovim.mode == normal && editorFocus",
	},
``` 

Hope this helps!
